src/command.rs

```diff
@@ -518,6 +518,86 @@ pub struct Credential<'l> {
     pub password_safe: Option<PasswordSafeData<'l>>,
 }
 
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Default)]
+pub struct CredentialUpdate<'l> {
```

Comment:
nit: why not `UpdateCredential`?  For the other commands, the structs and the `Command` variants use the same name.

Reply:
Yup, I saw that too, but too late. I can make a quickfix later.

Reply:
Done